logger/logger.go

```diff
@@ -0,0 +1,94 @@
+package logger
+
+import (
+	"context"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+var sugaredLogger *zap.SugaredLogger
+
+func SetupLogger(env string) (*zap.SugaredLogger, error) {
+	logger, err := getLoggerbyEnv(env)
+	if err != nil {
+		return nil, err
+	}
+
+	sugaredLogger = logger.Sugar()
+	return sugaredLogger, nil
+}
+
+func getLoggerbyEnv(env string) (logger *zap.Logger, err error) {
+	option := zap.AddCallerSkip(1)
+
+	if env == "production" {
```

Comment:
please move it to const and how about if no env is stated we start in production env